pkgs/development/libraries/gettext/default.nix

```diff
@@ -52,7 +52,7 @@ stdenv.mkDerivation rec {
     xz
     xz.bin
   ];
-  buildInputs = [ bash ]
+  buildInputs = lib.optionals (!stdenv.hostPlatform.isWindows) [ bash ]
```

Comment:
can be dropped https://github.com/NixOS/nixpkgs/commit/7bc650eccac8ee2d032a5beee6d44ff18527f7b6